src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/PemEncoding.cs

```diff
@@ -0,0 +1,290 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+#if NET
+#error This PemEncoding implementation is intended to be used only from downlevel targets. .NET should use the in-box implementation.
+#endif
+
+namespace System.Security.Cryptography
+{
+    // Downlevel implementation of PemEncoding. This was originally taken from the .NET 5 implementation of PemEncoding
+    // https://github.com/dotnet/runtime/blob/4aadfea70082ae23e6c54a449268341e9429434e/src/libraries/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/PemEncoding.cs
+    internal static class PemEncoding
```

Comment:
This is basically a direct clone of the .NET 5 implementation from `PemEncoding`, with the exception of being `internal` the XML doc comments are stripped out, and not having the `Write` methods (yet).